packages/appstore/src/types/common.types.ts

```diff
@@ -53,15 +53,15 @@ export type TJurisdictionData = Record<
 >;
 
 export type TDetailsOfEachMT5Loginid = DetailsOfEachMT5Loginid & {
+    account_id?: string;
     display_login?: string;
-    landing_company_short?: string;
```

Comment:
Removed `landing_company_short` because we already have this property in DetailsOfEachMT5Loginid type